website/src/components/canvas/Sidebar.jsx

```diff
@@ -89,7 +112,18 @@ class Sidebar extends React.Component {
   mouseEnter() {
     this.setState({opacity: 1})
   }
-
+  onClickClose() {
+   this.setState(prevState => ({
+     isOpen: !prevState.isOpen,
+     closed: 1
+   }));
+ }
+ onClickOpen() {
```

Comment:
Right, that's related to writing a toggle class though.
I guess my concern was mainly a naming thing since rather than having one toggle handler like they do, you have two open & close handlers -- in which case, if the roles were switched the handlers wouldn't perform what they are described to. I'm cool with !prevState if used in the right context!